### PR TITLE
Add min and max procedures to u64 stdlib module 

### PR DESCRIPTION
**Pull request by tohrnii**
Addresses Issue #172

### CODE REVIEW

File: stdlib/asm/math/u64.masm [hunk 1]
```diff
@@ -326,6 +326,65 @@ export.checked_eqz
...
+    dup.3
+    exec.unchecked_gt
+    dup.0
+    movup.4
+    movup.3
+    movup.2
+    cdrop
+    movdn.3
+    cdrop
+    swap
```
Comment (bobbinth) [inline stdlib/asm/math/u64.masm:346]: You may be able to save a couple of cycles by replacing these with:
```
movup.4
movup.3
dup.2
cdrop
movdn.3
cdrop
```

File: stdlib/asm/math/u64.masm [hunk 2]
```diff
@@ -326,6 +326,65 @@ export.checked_eqz
     and
 end
 
+# Performs greater-than comparison of two unsigned 64 bit integers and returns min value.
```
Comment (bobbinth) [inline stdlib/asm/math/u64.masm:329]: nit: prefer this to be "Compares two unsigned 64 bit integers and drop the larger one from the stack."

File: stdlib/asm/math/u64.masm [hunk 3]
```diff
@@ -326,6 +326,65 @@ export.checked_eqz
...
+    dup.3
+    exec.unchecked_lt
+    dup.0
+    movup.4
+    movup.3
+    movup.2
+    cdrop
+    movdn.3
+    cdrop
+    swap
```
Comment (bobbinth) [inline stdlib/asm/math/u64.masm:375]: You can probably save a couple of cycles by using the approach mentioned in the other comment.

**Review by bobbinth — CHANGES_REQUESTED** [review]
Looks good! Thank you for working on this!

I left a few comments inline - all of them are pretty minor.

**Comment by tohrnii** [comment]
Thank you. Made the suggested changes.

**Review by bobbinth — APPROVED** [review]
Looks great! Thank you!

Would you mind squashing the commits? And then I'll merge.

**Pull request merged**